install_config/master_node_configuration.adoc

```diff
@@ -798,6 +798,23 @@ networkConfig:
 
 |===
 
+.Pod and Node Configuration Parameters
+[cols="3a,6a",options="header"]
+|===
+
+| Parameter Name | Description
+
+|`*PodEvictionTimeout*`
+|Controls grace period for deleting pods on failed nodes. It takes valid time
+duration string. If empty, you get the default pod eviction timeout.
```

Comment:
@mburke5678 hi, I we need to move `PodEvictionTimeout` to master configuration https://docs.openshift.com/container-platform/3.11/install_config/master_node_configuration.html#master-node-config-kubernetes-master-config

Reply:
also include statement that the default value is 5 minutes. Default: 5m0s

Reply:
@sunilcio Does the `ProxyClientInfo` parameter also need to move to this table? The example master config in the BZ shows the variable.
https://bugzilla.redhat.com/show_bug.cgi?id=1770891#c1

Reply:
@mburke5678 yes, I see it in master config. @rphillips hi, could you please confirm, thanks!